devices/CMakeLists.txt

```diff
@@ -8,3 +8,8 @@ add_subdirectory(IAnalogSensorToIWear)
 if(XSENS_MVN_USE_SDK)
   add_subdirectory(XsensSuit)
 endif()
+
+if(iDynTree_DIR)
```

Comment:
What is this variable? Maybe `iDynTree_FOUND`?  @Yeshasvitvs @lrapetti

Reply:
done in https://github.com/robotology/wearables/pull/25/commits/32747cb9a99ee46d04b583a0dde3d39f36a7f40e